layers/transportation/update_transportation_merge.sql

```diff
@@ -171,17 +175,35 @@ CREATE INDEX IF NOT EXISTS osm_transportation_merge_linestring_gen_z7_geometry_i
 DROP MATERIALIZED VIEW IF EXISTS osm_transportation_merge_linestring_gen_z6 CASCADE;
 CREATE MATERIALIZED VIEW osm_transportation_merge_linestring_gen_z6 AS
 (
-SELECT ST_Simplify(geometry, ZRes(10)) AS geometry,
+SELECT ST_Simplify(geometry, ZRes(11)) AS geometry,
```

Comment:
why did this 10 change to an 11?

Reply:
This was to fix the new gaps that were being added - but I've now tried changing it back to 10 and the gaps aren't there. (maybe another change has fixed this or there was something I hadn't accounted for...)

Changing back to 10